modules/compute/vm-instance/variables.tf

```diff
@@ -73,6 +73,15 @@ variable "name_prefix" {
   default     = null
 }
 
+variable "always_include_deployment_name" {
+  description = <<-EOT
+    If true, the deployment name will always be included as part of the VM name. 
```

Comment:
I think here you could show how the instance will be named if this variable is set to true, for instance: 
If true, the instances will be named with both the prefix and the deployment name. 
"deployment name"-"prefix"-<index>. 
One would argue that, in this case, it is a prefix and not postfix, but I think renaming the variable is a bad idea.

Reply:
The word you're looking for is "affix")

Reply:
If we want to avoid "see README", I suggest the following description changes...

From

```hcl
variable "always_include_deployment_name" {
  description = <<-EOT
    If true, the deployment name will always be included as part of the VM name. 
    If false, it will only be included if `name_prefix` is not provided.
    EOT
  type        = bool
  default     = false
}
```

to

> If true, the names of VMs and disks will always be prefixed with var.deployment_name to enable uniqueness across deployments. See var.name_prefix for further details on resource naming behavior.

I also suggest changing:

```hcl
variable "deployment_name" {
  description = "Name of the deployment, used to name the cluster"
  type        = string
}
```

to

> Name of the deployment, will optionally be used name resources according to var.name_prefix

and

```hcl
variable "name_prefix" {
  description = "Name Prefix"
  type        = string
  default     = null
}
```

> An optional name for all VM and disk resources. If not supplied, var.deployment_name will be used. When var.name_prefix is supplied, and var.always_include_deployment_name is set, then resources are named by \"var.deployment_name-var.name_prefix\".

Reply:
Updated descriptions as suggested.